src/AMAMM.sol

```diff
@@ -56,7 +56,8 @@ contract AMAMM is IAmAmm {
     mapping(PoolId id => mapping(uint40 => Bid)) public poolEpochManager;
     mapping(address manager => mapping(PoolId id => uint256)) internal _refunds;
     mapping(address manager => mapping(Currency currency => uint256)) internal _fees;
-    mapping(PoolId id => mapping(uint40 => mapping(address => Bid))) public poolEpochBids;
+    mapping(address deposits => uint256) public _userDeposits;
+    mapping(address refunds => uint256) public _userRefunds;
```

Comment:
I think we don't need userRefunds as could just reuse userDeposits and just increment the amount as refund.